core/src/services/swift/core.rs

```diff
@@ -270,10 +270,11 @@ mod tests {
                 bytes: 147,
                 hash: "5e6b5b70b0426b1cc1968003e1afa5ad".to_string(),
                 name: "test.txt".to_string(),
-                content_type:
-                    "multipart/form-data;boundary=------------------------25004a866ee9c0cb"
-                        .to_string(),
                 last_modified: "2023-11-01T03:00:23.147480".to_string(),
+                content_type: Some(
+                    "multipart/form-data;boundary=------------------------25004a866ee9c0cb"
```

Comment:
Should this be `text/plain`?

Reply:
`multipart/form-data` seems misleading here, we can use `application/octet-stream` instead, and `text/plain` should also work.

Reply:
Yes, here is a wrong test, I changed it to `text/plain`.